pandas/tests/series/test_analytics.py

```diff
@@ -1015,6 +1015,11 @@ def test_unique(self):
         tm.assert_categorical_equal(s.unique(), pd.Categorical([np.nan]),
                                     check_dtype=False)
 
+    def test_unique_obj_na_preservation(self, nulls_fixture):
+        # GH 20866
+        s = pd.Series(['foo', nulls_fixture])
+        assert s.iloc[1] is nulls_fixture
```

Comment:
Looks like the `.unique()` is missing from `s`.

Reply:
Ah, that's a good reason that it is passing :-)

Reply:
Whoops...hmm well yes now I am getting `nan` just as you are so something strange is going on here. Going to bisect between d274d0b22ec3fa0048b27df7b5f349f1779a8c3f and 9736ecb63e66828a3f814146c6e9679725a24c2e as the behavior changed for me between those. Will post back results soon